great_expectations/rule_based_profiler/data_assistant/onboarding_data_assistant.py

```diff
@@ -785,3 +790,138 @@ def _build_categorical_rule() -> Rule:
         )
 
         return rule
+
+    @staticmethod
+    def _build_text_rule() -> Rule:
+
+        # Step-1: Instantiate "ColumnDomainBuilder" for selecting proper text columns.
+
+        text_column_type_domain_builder: ColumnDomainBuilder = ColumnDomainBuilder(
+            include_column_names=None,
+            exclude_column_names=None,
+            include_column_name_suffixes=None,
+            exclude_column_name_suffixes=None,
+            semantic_type_filter_module_name=None,
+            semantic_type_filter_class_name=None,
+            include_semantic_types=[
+                SemanticDomainTypes.TEXT,
+            ],
+            exclude_semantic_types=None,
+            data_context=None,
+        )
+
+        # Step-2: Declare "ParameterBuilder" for every metric of interest.
+
+        column_min_length_range_estimator_parameter_builder: NumericMetricRangeMultiBatchParameterBuilder = NumericMetricRangeMultiBatchParameterBuilder(
+            name="column_min_length_range_estimator",
+            metric_name="column_values.length.min",
+            metric_domain_kwargs=DOMAIN_KWARGS_PARAMETER_FULLY_QUALIFIED_NAME,
+            metric_value_kwargs=None,
+            enforce_numeric_metric=True,
+            replace_nan_with_zero=True,
+            reduce_scalar_metric=True,
+            false_positive_rate=f"{VARIABLES_KEY}false_positive_rate",
+            quantile_statistic_interpolation_method=f"{VARIABLES_KEY}quantile_statistic_interpolation_method",
+            estimator=f"{VARIABLES_KEY}estimator",
+            n_resamples=f"{VARIABLES_KEY}n_resamples",
+            random_seed=f"{VARIABLES_KEY}random_seed",
+            include_estimator_samples_histogram_in_details=f"{VARIABLES_KEY}include_estimator_samples_histogram_in_details",
+            truncate_values=f"{VARIABLES_KEY}truncate_values",
+            round_decimals=f"{VARIABLES_KEY}round_decimals",
+            evaluation_parameter_builder_configs=None,
+            json_serialize=True,
+        )
+        column_max_length_range_estimator_parameter_builder: NumericMetricRangeMultiBatchParameterBuilder = NumericMetricRangeMultiBatchParameterBuilder(
```

Comment:
@cdkini This might be in error -- metrics parameter builders' names end on `_for_metrics` and they are probably never of the "range" type.  Thanks.

Reply:
updated!